otelcol/command.go

```diff
@@ -42,7 +42,7 @@ func NewCommand(set CollectorSettings) *cobra.Command {
 	rootCmd.AddCommand(newFeatureGateCommand())
 	rootCmd.AddCommand(newComponentsCommand(set))
 	rootCmd.AddCommand(newValidateSubCommand(set, flagSet))
-	rootCmd.AddCommand(newConfigPrintSubCommand(set, flagSet))
+	rootCmd.AddCommand(newConfigPrintSubCommand(set, flagSet, os.Stdout, os.Stderr))
```

Comment:
if passing the os.Std[err/out] is for testing, I would rather have a different `newConfigPrintSubCommandInternal` that accepts that and `newConfigPrintSubCommand` does not accept them.